engine/src/main/java/org/camunda/bpm/engine/TelemetryService.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.engine;
+
+import org.camunda.bpm.engine.impl.telemetry.dto.TelemetryData;
+
+/**
+ * Service which provides access to data collected for telemetry.
+ * Telemetry data is also available if you did not opt-in to share the data with Camunda.
+ */
+public interface TelemetryService {
```

Comment:
I recommend to not add a new service for this feature. We can use services to group common functionality together to make our API easier to discover. I think this is not justified here because:

* The service has a single method
* `ManagementService` already has telemetry-related and metrics-related methods. There is no clear distinction for users to know what is where.
* Services are fairly heavy API constructs, users need to know and learn about them in order to find the API methods. It's only a good idea if we have a strong motivation to create a new service (e.g. a lot of common functionality). I do not see this here.